x/staking/keeper/keeper.go

```diff
@@ -41,6 +42,49 @@ func HistoricalInfoCodec(cdc codec.BinaryCodec) collcodec.ValueCodec[types.Histo
 	})
 }
 
+type moduleValidatorUpdatesCodec struct {
```

Comment:
@testinginprod, can we generalize that in collection? Like a JSONValueCodec that always does the following?

Reply:
It isn't needed here, but I'll create one now in collections